src/apps/common/ns_turn_utils.c

```diff
@@ -147,6 +147,48 @@ int turn_mutex_destroy(turn_mutex* mutex) {
 
 ///////////////////////// LOG ///////////////////////////////////
 
+/* syslog facility */
+/*BVB-594  Syslog facility */
+static char* str_fac[]={"LOG_AUTH","LOG_CRON","LOG_DAEMON",
+			"LOG_KERN","LOG_LOCAL0","LOG_LOCAL1",
+			"LOG_LOCAL2","LOG_LOCAL3","LOG_LOCAL4","LOG_LOCAL5",
+			"LOG_LOCAL6","LOG_LOCAL7","LOG_LPR","LOG_MAIL",
+			"LOG_NEWS","LOG_USER","LOG_UUCP",
+			"LOG_AUTHPRIV","LOG_FTP","LOG_SYSLOG",
+			0};
+
+static int int_fac[]={LOG_AUTH ,  LOG_CRON , LOG_DAEMON ,
+		    LOG_KERN , LOG_LOCAL0 , LOG_LOCAL1 ,
+		    LOG_LOCAL2 , LOG_LOCAL3 , LOG_LOCAL4 , LOG_LOCAL5 ,
+		    LOG_LOCAL6 , LOG_LOCAL7 , LOG_LPR , LOG_MAIL ,
+		    LOG_NEWS , LOG_USER , LOG_UUCP,
+		    LOG_AUTHPRIV,LOG_FTP,LOG_SYSLOG,
+		    0};
+
+static int syslog_facility = 0;
+
+static int str2facility(char *s)
```

Comment:
can you rename to `str_to_syslog_facility` (for consistency with other functions like `addr_to_string`)?

Reply:
I realized some incomplete coding in my previous commit and fixed them.
Thanks for the comment.